Bottom: 9e98fc86d59414946590790810e290822966624a
Top:    a3988a5bc55f6781f9f3e69c59b0896af194a3aa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 00:34:43 +0100

Refresh of draw-c-split-gathering-from-re

---

diff --git a/src/draw.c b/src/draw.c
index 5c34b25..2aeefc1 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1150,8 +1150,6 @@ DrawArcLowLevel (ArcTypePtr Arc)
 static void
 DrawElementPackageLowLevel (ElementTypePtr Element)
 {
-  assert (Gathering);
-
   /* draw lines, arcs, text and pins */
   ELEMENTLINE_LOOP (Element);
   {
@@ -1496,6 +1494,8 @@ DrawPlainPolygon (LayerTypePtr Layer, PolygonTypePtr Polygon)
 void
 DrawElement (ElementTypePtr Element)
 {
+  assert (Gathering);
+
   DrawElementPackage (Element);
   DrawElementName (Element);
   DrawElementPinsAndPads (Element);
